### PR TITLE
API-4149: Play 2.6 upgrade

### PR DESCRIPTION
**Pull request by mattclark-zerogravit**

### CODE REVIEW

File: conf/logback.xml [hunk 1]
```diff
@@ -35,17 +35,18 @@
...
-    </logger>
-
-    <logger name="com.ning.http.client.providers.netty" additivity="false">
-        <appender-ref ref="STDOUT_IGNORE_NETTY" />
-    </logger>
-
-    <logger name="com.google.inject" level="INFO"/>
-    <logger name="org.asynchttpclient.netty.channel.DefaultChannelPool" level="INFO"/>
-    <logger name="io.netty" level="INFO"/>
+<!--    <logger name="accesslog" level="INFO" additivity="false">-->
```
Comment (lburgos) [inline conf/logback.xml]: Do we need this?

File: build.sbt [hunk 2]
```diff
@@ -53,11 +84,11 @@ lazy val microservice = (project in file("."))
     targetJvm := "jvm-1.8",
     scalaVersion := "2.11.11",
     libraryDependencies ++= appDependencies,
+    //dependencyOverrides ++= overrides,
```
Comment (lburgos) [inline build.sbt]: don't we need the overrides in the end?

File: test/stubs/WireMockSupport.scala [hunk 3]
```diff
@@ -0,0 +1,60 @@
...
+
+  override protected def beforeAll(): Unit = {
+    super.beforeAll()
+    WireMock.configureFor("localhost", mockServerPort)
+    mockServer.start()
+  }
+
+  override protected def beforeEach(): Unit = {
+    super.beforeEach()
+    //sy
```
Comment (lburgos) [inline test/stubs/WireMockSupport.scala]: we could delete this

File: conf/logback.xml (hunk 1 above)
Comment (mattclark-zerogravit) [reply conf/logback.xml]: oops... no I'll add back in. Was messing with the logs to try and see why something wasn't happening.

Comment (mattclark-zerogravit) [reply conf/logback.xml]: will fix up

File: test/stubs/WireMockSupport.scala (hunk 3 above)
Comment (mattclark-zerogravit) [reply test/stubs/WireMockSupport.scala]: will fix up

File: build.sbt (hunk 2 above)
Comment (mattclark-zerogravit) [reply build.sbt]: yep have re added

**Pull request merged**